components/submissions/SubmissionsTable.tsx

```diff
@@ -1,30 +1,40 @@
-import { StatusBadge, StatusEnum } from "./StatusBadge";
+import { FC } from "react";
+import { useRouter } from "next/router";
+import { Pagination, Submission } from "../../interfaces";
+import { StatusBadge } from "./StatusBadge";
+import { ChevronIcon } from "../icons/Icons";
+import moment from "moment";
 
-const rows = [
-  {
-    id: 1,
-    title: "Hepatic Infraction",
-    doctor: "",
-    createdAt: "22/02/2021",
-    status: StatusEnum.Pending,
-  },
-  {
-    id: 2,
-    title: "Pancreatis Acute",
-    doctor: "Dr. John Doe",
-    createdAt: "22/02/2021",
-    status: StatusEnum.InProgress,
-  },
-  {
-    id: 3,
-    title: "Bone marrow failure",
-    doctor: "Dr. John Doe",
-    createdAt: "22/02/2021",
-    status: StatusEnum.Done,
-  },
-];
+interface SubmissionsTableProps {
+  submissions: Submission[];
+  pagination: Pagination;
+  changeStatus: (status: string) => void;
+  changePage: (page: number) => void;
+}
+
+const getDateFormat = (date: Date) => {
+  let dateMoment = moment(date);
```

Comment:
const

Reply:
also I JUST noticed you're using moment. I would advice you to use date-fns instead since its api call are non-mutating, unlike moment where you're basically needing to create new date objects all the time to avoid mutations